### ISSUE TITLE
Add an error assert task to the scenario player.  

### ISSUE THREAD

**Issue by err508** [raiden-network/raiden#2579]
The he scenario player currently does not allow to test that something fails as expected, i.e. opening a channel twice.

**Comment by hackaugusto** [comment]
Adding the expected HTTP response code should be sufficient IMO

**Comment by CosminNechifor** [comment]
#2554

**Comment by ulope** [comment]
Yeah,  I agree with Rakan for the API-calls-should-fail cases. As discussed before adding a field for the expected error code is more flexible and less prone to false negatives.

However, having a generic Exception catching Task could still be useful for other cases.
